taskcluster/ci/complete/kind.yml

```diff
@@ -14,8 +14,15 @@ transforms:
     - taskgraph.transforms.code_review:transforms
```

Comment:
Oh, maybe this transform already solves https://github.com/taskcluster/taskgraph/issues/71 :). I guess we could rename it and make it a bit more generic still.

Reply:
Oh yeah! 😃 I remember Tom Prince imported it in Taskgraph. The name was already set in gecko, IIRC.